xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp

```diff
@@ -181,16 +179,38 @@ static int AEChannelMapToAUDIOTRACKChannelMask(CAEChannelInfo info)
 jni::CJNIAudioTrack *CAESinkAUDIOTRACK::CreateAudioTrack(int stream, int sampleRate, int channelMask, int encoding, int bufferSize)
 {
   jni::CJNIAudioTrack *jniAt = NULL;
-  m_jniAudioFormat = encoding;
 
   try
   {
-    jniAt = new CJNIAudioTrack(stream,
-                               sampleRate,
-                               channelMask,
-                               encoding,
-                               bufferSize,
-                               CJNIAudioTrack::MODE_STREAM);
+    if (CJNIBase::GetSDKVersion() >= 21)
+    {
+      CJNIAudioAttributesBuilder attrBuilder;
+      attrBuilder.setContentType(stream == CJNIAudioManager::STREAM_MUSIC
+        ? CJNIAudioAttributes::CONTENT_TYPE_MUSIC
+        : CJNIAudioAttributes::CONTENT_TYPE_MOVIE);
```

Comment:
Do you ever want to use CONTENT_TYPE_MOVIE?

Reply:
I want either MUSIC or MOVIE, as written in code

Reply:
https://github.com/peak3d/xbmc/blob/e22caaf2d862c72be6672de124f5111541d376cd/xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp#L305 <- that's the only place it is called and this is hard coded to MUSIC that's why I asked.